app/src/main/java/com/elavon/converge/config/Config.java

```diff
@@ -34,6 +43,37 @@ public void setLog(Log log) {
         this.log = log;
     }
 
+    public static class Credential {
```

Comment:
converge credentials coming from resource file for now. will come from some other source in the future.